infor-design/enterprise
# 6086 - Modal Focus Issue Hidden Elements

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR will fix hidden focusable item when gets shown is still being disabled.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/6086

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app

Initial Issue
- Go to http://localhost:4000/components/modal/test-toggle-disabled.html
- Click `Show Modal`
- Click `Test`
- See the `Input` is now visible and clickable.

Secondary Issue (#5875)
- Go to http://localhost:4000/components/modal/example-hidden-field.html
- Click `Show Modal`
- Tab around the modal - should skip the hidden field

**Included in this Pull Request**:
